packages/generator/templates/ClientInstance.ts.hbs

```diff
@@ -1,15 +1,9 @@
-
+/* eslint-disable */
```

Comment:
Why do we need to disable eslint?

Reply:
We shouldn't here, I would say this needs to be removed from this file

Reply:
This is WIP, as noted in my other comment, this will be removed prior to merge. It was here to stop be getting so much red in VSCode